src/server/utilities/cspHeader/index.js

```diff
@@ -76,6 +80,9 @@ const directives = {
       'https://polling.bbc.co.uk', // Media page
       'https://securepubads.g.doubleclick.net', // ads
       'https://tpc.googlesyndication.com', // ads
+      'https://www.youtube.com', // Social Embeds, <amp-youtube />
+      'https://www.instagram.com', // Social Embeds, <amp-instagram />
+      'https://*.ampproject.net', // Social Embeds
```

Comment:
This array is for the Canonical page on Test, so you don't need this amp domain here
```suggestion
```